src/Controllers/LoginController.php

```diff
@@ -101,8 +101,12 @@ protected function getValidationRules(): array
      */
     public function logoutAction(): RedirectResponse
     {
+        // Capture logout redirect URL before auth logout,
+        // otherwise 'Call to a member function inGroup() on null' is thrown
```

Comment:
How about this?

```suggestion
        // Capture logout redirect URL before auth logout,
        // otherwise you cannot check the user in `logoutRedirect()`.
```

Reply:
Good suggestion.
@cornejobarraza  Apply this Suggested by clicking on button `Commit suggestion`.